Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -875,6 +875,18 @@ def genS(fragment,howMuch):
 step1FastUpg2015_trackingOnlyValidation = merge([{'-s':'GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,DIGI2RAW,RECO,VALIDATION:@trackingOnlyValidation'},
                                                 step1FastUpg2015Defaults])
 
+step1FastUpg2017Defaults =merge([{'-s':'GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,DIGI2RAW,L1Reco,RECO,EI,VALIDATION:@standardValidation,DQM:@standardDQM',
+                           '--fast':'',
+                           '--conditions'  :'auto:phase1_2017_realistic',
+                           '--beamspot'    : 'Realistic50ns13TeVCollision',
```

Comment:
@angirar : note that in Phase 1 MC the beasmpot is Realistic25ns13TeVEarly2017Collision (you can check the fullsim relvals)

Reply:
Made the change. Thanks.